### ISSUE TITLE
Checkbox is not visible in detailsList once screen reader is on in scan mode

### ISSUE THREAD

**Issue by Harkesh030679** [microsoft/fluentui#13287]
Checkbox is not visible in DetailsList once screen reader is on in scan mode

### Environment Information

OS: Win 10
Browser: Edge

### Describe the issue:

Checkbox is not visible in DetailsList once screen reader is on in scan mode

### Please provide a reproduction of the issue in a codepen:

Open below URL
https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/customitemcolumns
(Rendering custom item columns with sorting)
Steps to repro
1. Press cntr+win+enter to on  screen reader
2. Press Capslock+ spacebar to on Scan mode
3. Now move on DetailsList item through Up/Down Arrow keys and see the issue
 as below snaps
Checkbox Uncheck--
![DetailList_Uncheck](https://user-images.githubusercontent.com/62233836/82660545-317bd980-9c48-11ea-9920-250b95411960.png)
Checkbox Checked
![DetailList_check](https://user-images.githubusercontent.com/62233836/82660600-48bac700-9c48-11ea-8f59-da1b77513d80.png)

#### Actual behavior:

Checkbox is not visible in detailsList once screen reader is on in scan mode

#### Expected behavior:

Checkbox should visible in detailsList once screen reader is on in scan mode

**Comment by aneeshack4** [comment]
Scan mode is known to be quite buggy... but I see the unchecked checkbox quite clearly:
![image](https://user-images.githubusercontent.com/4161791/82690505-f8c90980-9c21-11ea-8c14-cbcf013542e9.png)
This control is aria appropriate as far as I can tell. Unless there are some aria gaps we can fill, this isn't actionable on our end.

**Comment by xugao** [comment]
We have a bug to track this: https://microsoft.visualstudio.com/Edge/_workitems/edit/24808403
I will ping on this
